<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsOWSSourceWidgetBase</class>
 <widget class="QgsProviderSourceWidget" name="QgsOWSSourceWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XYZ Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="mTimeWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mTimeLabel">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mTimeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mCRSWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mCRSLabel">
        <property name="text">
         <string>Coordinate Reference System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mSelectedCRSLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Selected Coordinate Reference System</string>
        </property>
        <property name="buddy">
         <cstring>mChangeCRSButton</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mChangeCRSButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Changeâ€¦</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mFormatWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mFormatLabel">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mFormatComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mWMSGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="mTileWidthLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mTileSizeLabel">
        <property name="text">
         <string>Tile size</string>
        </property>
        <property name="buddy">
         <cstring>mTileWidthLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="mLayerNameLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="mTileHeightLineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="mFeatureCountLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mLayerNameLabel">
        <property name="text">
         <string>Layer name</string>
        </property>
        <property name="buddy">
         <cstring>mLayerNameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="mFeatureCountLabel">
        <property name="text">
         <string>Feature limit for GetFeatureInfo</string>
        </property>
        <property name="buddy">
         <cstring>mFeatureCountLineEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mCacheWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mCacheLabel">
        <property name="text">
         <string>Cache</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mCacheComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Cache preference

Always cache: load from cache, even if it expired

Prefer cache: load from cache if available, otherwise load from network. Note that this can return possibly stale (but not expired) items from cache

Prefer network: default value; load from the network if the cached entry is older than the network entry

Always network: always load from network and do not check if the cache has a valid entry (similar to the &quot;Reload&quot; feature in browsers)
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mSpatialExtentBoxWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QgsExtentGroupBox" name="mSpatialExtentBox">
        <property name="title">
         <string>Extent (current: none)</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="collapsed" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProviderSourceWidget</class>
   <extends>QWidget</extends>
   <header>qgsprovidersourcewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
